example/src/main/java/org/folio/tlib/example/Book.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.tlib.example;
```

Comment:
Cosmetics: move to `org.folio.tlib.example.do` (for data objects) or `org.folio.tlib.example.data`